### PR TITLE
new sub heading created

### PR DESCRIPTION
**Pull request by HsnAkk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,9 @@
     <nav>
         <h1>Heading from Hasan</h1>
     </nav>
+    <main>
+        <div>This is new subHeading</div>
```
Comment (ghassanaldarwish) [inline index.html]: Please replace the div element with h2 elemant

**Review by ghassanaldarwish — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (HsnAkk) [reply index.html]: I corrected the code content, please check it

**Review by ghassanaldarwish — APPROVED** [review]
Good job!

File: index.html (hunk 1 above)
Comment (ghassanaldarwish) [reply index.html]: Thanks Hasan

**Pull request merged**